src/telliot_feeds/cli/commands/report.py

```diff
@@ -256,13 +182,44 @@ def reporter() -> None:
     prompt=False,
 )
 @click.option(
-    "--custom-contract",
-    "-custom",
-    "custom_contract_reporter",
```

Comment:
If someone has a custom contract that they want to use to report or wrap any of the oracle, token, or autopay contracts used by 360/flex, can't we just sub out the addresses if their custom contract has the appropriate ABI? This is why I don't think the flag is needed, but correct me if I'm misunderstanding what the original intention was!

Reply:
I think I'm going to add prompts for the ABI instead of a flag, and then the `verify_custom_contract_abi` which will reuse some of the "abi auto fetching and abi not found msg" code you wrote probs

Reply:
For using a custom contract, the abi could be different but a prompt is a good idea and also one thing to keep in mind is that the contract is the reporter and not the account(contract owner).